services/app-api/scripts/import-programs.js

```diff
@@ -0,0 +1,123 @@
+const csv = require('csv-parser');
+const fs = require('fs');
+const slugify = require("slugify");
+const { v5: uuidv5 } = require('uuid');
+
+const stateNames = {
```

Comment:
I’m going to add American Samoa to our google sheet b/c we use it for testing, AS will need to be added here.